ar4_common/README.md

```diff
@@ -5,7 +5,7 @@ To build the package, run
 `colcon build`
 
 To run the simulation, source and run
-`ros2 launch ar4_gazebo ar4_sim.launch.py`
+`ros2 launch ar4_common ar4_sim.launch.py`
```

Comment:
Is this correct? If that's the case, it is unclear what simulation we are launching here.

Reply:
Not really. I did not see there was a readme in this folder. The documentation in the source folder should suffice.
I deleted it.
Good catch.